### PR TITLE
Bug 1199622 Update partner requirement PDF links

### PR DESCRIPTION
**Pull request by sgarrity**

### CODE REVIEW

File: bedrock/mozorg/templates/mozorg/contentservices/faq.html [hunk 1]
```diff
@@ -152,8 +152,8 @@ <h3 data-accordion-role="tab">{{ _('What input do I have over the interest categ
     <h3 data-accordion-role="tab">{{ _('What are the technical requirements?') }}</h3>
     <div data-accordion-role="tabpanel">
       <p>
-        {% trans url='http://static.mozilla.com/moco/en-US/pdf/Partner_Specs_Directory_Tiles.pdf', url2='http://static.mozilla.com/moco/en-US/pdf/Partner_Specs_Suggested_Tiles.pdf' %}
-          Please see our our creative guidelines for <a href="{{ url }}">Directory Tiles</a> or <a href="{{ url2 }}">Suggested Tiles</a> for guidance on how to create Tile assets.
+        {% trans url='http://static.mozilla.com/moco/en-US/pdf/Creative_Specs_for_Directory_Suggested_Tiles.pdf' %}
+          Please see our our creative creative guidelines for <a href="{{ url }}">Directory Tiles and Suggested Tiles</a> for guidance on how to create Tile assets.
```
Comment (schalkneethling) [inline bedrock/mozorg/templates/mozorg/contentservices/faq.html]: `our creative` is duplicated.

Comment (sgarrity) [inline bedrock/mozorg/templates/mozorg/contentservices/faq.html]: > `our creative` is duplicated.

Fixed, thanks.

File: bedrock/mozorg/templates/mozorg/contentservices/faq.html [hunk 2]
```diff
@@ -152,8 +152,8 @@ <h3 data-accordion-role="tab">{{ _('What input do I have over the interest categ
     <h3 data-accordion-role="tab">{{ _('What are the technical requirements?') }}</h3>
     <div data-accordion-role="tabpanel">
       <p>
-        {% trans url='http://static.mozilla.com/moco/en-US/pdf/Partner_Specs_Directory_Tiles.pdf', url2='http://static.mozilla.com/moco/en-US/pdf/Partner_Specs_Suggested_Tiles.pdf' %}
-          Please see our our creative guidelines for <a href="{{ url }}">Directory Tiles</a> or <a href="{{ url2 }}">Suggested Tiles</a> for guidance on how to create Tile assets.
+        {% trans url='http://static.mozilla.com/moco/en-US/pdf/Creative_Specs_for_Directory_Suggested_Tiles.pdf' %}
+          Please see our our creative guidelines for <a href="{{ url }}">Directory Tiles and Suggested Tiles</a> for guidance on how to create Tile assets.
```
Comment (schalkneethling) [inline bedrock/mozorg/templates/mozorg/contentservices/faq.html]: Still to "our"s :)

Comment (sgarrity) [inline bedrock/mozorg/templates/mozorg/contentservices/faq.html]: > Still two "our"s :)

Ouch... sorry. I think I've got it this time.

**Comment by schalkneethling** [comment]
r+ :cake:

**Pull request merged**